packages/terra-framework-docs/src/terra-dev-site/doc/pills/FilterPills.1.doc.mdx

```diff
@@ -0,0 +1,122 @@
+| react-intl | >=2.8.0 <6.0.0 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+```jsx
+import FilterPills from 'terra-pills';
+```
+
+## Component Features
+
+* [Cross-Browser Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#cross-browser-support)
+* [Responsive Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#responsive-support)
+* [Mobile Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#mobile-support)
+
+# Introduction
+Filter Pills can be added to a variety of situations and compositions to apply context of the applied categories or labels. They can be constructed
+into in various higher-order component patterns like an ʻApplied Filters' toolbar, or locations that the applied categery labels are removable by the
+user. They also have the ability to flex layouts between an expanded state showing all filter pills, and a collapsed state showing just a single line
+and roll-up toggle to show and hide the additional filter pills.
+
+<AppliedFiltersExample title='Introduction to Filter Pills: Applied Filters Toolbar' />
+
+# How To Construct:
+
+## Filter Pill with basic Label
+
+Filter Pills in their standard form display a single keyword or short phrase as their label. They are also removeable, which makes them interactive
+with the clickable remove ("X") button, as well as by interactive by keyboard, able to be removed by pressing the 'delete' or 'backspace' key when
+they are in focus.
+
+<BasicLabelExample title="Basic Label" />
+
+## Adding an optional Category to the Label
+
+Additional category terms can be added to the single keyword or short phrase label to provide additional clarity in situations where the same
+keyword label may appear twice (but apply to different groupings), or is needed to help indicate to a user the category groupings the labels
+belong to.
+
+<CategoryLabelExample title="Adding a Label Category" />
+
+## Long Labels & Truncation: Popup is Added
+
+In cases where the width of the space is constrained and the text label for the Filter Pill is unable to fully display, a popup will automatically be
+added and the label will become an interactive target. Clicking the label will disclose the full Filter Pill text label (and category if applied) as the
+contents. No additional content can be added to the popup.
+
+<TruncatedLabelExample title="Selectable Label when Truncated" />
+
+## Adding a Visible Label
+
+Filter Pills are commonly used in a group or list, which must include a label to indicate the intent or purpose of the contents of the list.
+Accordingly, the filter pills component requires the `ariaLabel` prop for accessibility minimum standards, to give a screenreader user the proper
+context of the list, but it is not visible to users.
+
+For best practice, it is recommended that pills include a visible label, which may need to be styled and positioned according to a specific usage
```

Comment:
> it is recommended that pills include a visible label

This makes it sound like consumers are implementing a custom label for each pill

Reply:
This part still reads the same. I think it would be more clear as:

> For best practice, the Filter Pills component should be accompanied by a visible label, ...

Reply:
[updated](https://github.com/cerner/terra-framework/pull/1490/commits/8d775bb7b611089a8d939625bd2832781544462a#diff-c0f356f3f58def9dbe21de651bee38f73c27a6b729e3d1170e20542793c04538R22)